zombienet/0002-validators-warp-sync/test-validators-warp-sync.toml

```diff
@@ -0,0 +1,35 @@
+[settings]
+enable_tracing = false
```

Comment:
What is this doing?

Reply:
https://github.com/paritytech/zombienet/pull/582